modules/sdk/gradle-plugins-defaults/src/main/java/com/liferay/gradle/plugins/defaults/LiferayOSGiDefaultsPlugin.java

```diff
@@ -396,6 +398,8 @@ else if (FileUtil.exists(project, ".lfrbuild-tool")) {
 
 		_addTaskCopyLibs(project);
 
+		Task generatePomInfoTask = _addTaskGeneratePomInfo(project);
```

Comment:
Can't we add this after the `_addTaskDeployDependencies` task, to keep it sorted?